iconMappings.ts

```diff
@@ -0,0 +1,107 @@
+	dotsHorizontal: { file: 'dots-horizontal.svg', fa_icon: 'far-ellipsis' },
+	download: { file: 'download.svg', fa_icon: 'fas-download' },
+	dragVertical: { file: 'drag-vertical.svg', fa_icon: 'fasr-grip-dots-vertical' },
+	email: { file: 'email.svg', fa_icon: 'fas-envelope' },
+	exclamation: { file: 'exclamation.svg', fa_icon: 'fas-exclamation' },
+	eye: { file: 'eye.svg', fa_icon: 'fas-eye' },
+	eyeOutline: { file: 'eye-outline.svg', fa_icon: 'far-eye' },
+	facebook: { file: 'facebook.svg', fa_icon: 'fab-facebook' },
+	file: { file: 'file.svg', fa_icon: 'fas-file' },
+	fileCompare: { file: 'file-compare.svg', fa_icon: 'fas-file-plus-minus' },
+	fileDocument: { file: 'file-document.svg', fa_icon: null },
+	fileMultiple: { file: 'file-multiple.svg', fa_icon: 'fas-files' },
+	filePdfBox: { file: 'file-pdf-box.svg', fa_icon: null },
+	filePresentationBox: { file: 'file-presentation-box.svg', fa_icon: null },
+	fileTableBox: { file: 'file-table-box.svg', fa_icon: null },
+	fileUpload: { file: 'file-upload.svg', fa_icon: 'fas-file-arrow-up' },
+	filter: { file: 'filter.svg', fa_icon: 'fas-filter' },
+	folderMove: { file: 'folder-move.svg', fa_icon: null },
+	folderMultiple: { file: 'folder-multiple.svg', fa_icon: 'fas-folders' },
+	folderOpen: { file: 'folder-open.svg', fa_icon: 'fas-folder-open' },
+	folderPlus: { file: 'folder-plus.svg', fa_icon: 'fas-folder-plus' },
+	folder: { file: 'folder.svg', fa_icon: 'fas-folder' },
+	formatListBulletedSquare: { file: 'format-list-bulleted-square.svg', fa_icon: 'fasr-list' },
+	fullscreen: { file: 'fullscreen.svg', fa_icon: 'fas-expand' },
+	fullscreenExit: { file: 'fullscreen-exit.svg', fa_icon: 'fas-compress' },
+	headset: { file: 'headset.svg', fa_icon: 'fas-headphones' },
+	history: { file: 'history.svg', fa_icon: 'fa-clock-rotate-left' },
+	home: { file: 'home.svg', fa_icon: 'fas-house' },
+	image: { file: 'image.svg', fa_icon: 'fas-image' },
+	imageEdit: { file: 'image-edit.svg', fa_icon: null },
+	imageOffOutline: { file: 'image-off-outline.svg', fa_icon: 'far-image-slash' },
+	information: { file: 'information.svg', fa_icon: 'fas-circle-info' },
+	informationVariant: { file: 'information-variant.svg', fa_icon: 'fa-info' },
+	key: { file: 'key.svg', fa_icon: 'fas-key' },
+	laptop: { file: 'laptop.svg', fa_icon: 'fas-laptop' },
+	link: { file: 'link.svg', fa_icon: 'fas-link' },
+	listBox: { file: 'list-box.svg', fa_icon: 'fas-rectangle-list' },
+	lock: { file: 'lock.svg', fa_icon: 'fas-lock' },
+	lockOpen: { file: 'lock-open.svg', fa_icon: 'fas-lock-open' },
+	magnify: { file: 'magnify.svg', fa_icon: 'far-magnifying-glass' },
+	mapClock: { file: 'map-clock.svg', fa_icon: 'far-map-location-dot' },
+	mapMarker: { file: 'map-marker.svg', fa_icon: 'fas-location-dot' },
+	menuDown: { file: 'menu-down.svg', fa_icon: 'fas-caret-down' },
+	menuUp: { file: 'menu-up.svg', fa_icon: 'fas-caret-up' },
+	messageReplyText: { file: 'message-reply-text.svg', fa_icon: 'fas-messages-lines' },
```

Comment:
```suggestion
	messageReplyText: { file: 'message-reply-text.svg', fa_icon: 'fas-message-dots' },
```
imo this should be the propper mapping for FA icon name.

Reply:
This was just copied from vue-material-design. But if you want, I can still change it 
https://github.com/IONOS-Productivity/nc-vue-material-design-icons/blob/ionos-dev/mapping.ts#L68C5-L68C24

Reply:
Preliminary, I fixed the typo